onnxruntime/core/framework/session_state_utils.cc

```diff
@@ -396,6 +398,16 @@ common::Status SaveInputOutputNamesToNodeMapping(const onnxruntime::GraphViewer&
       // is consumed in the subgraph if there is an explicit consumer.
       // If the only consumer(s) are implicit consumers (i.e.) other control flow nodes, its
       // location is the location of the value in the enclosing outer scope.
+
+      // In the main graph, the location of the implicit input(s) is the location it
```

Comment:
Only change here

Reply:
Can we put main graph comment before subgraph comment?

Reply:
Sure